### PR TITLE
Update trivy to 0.37.3

### PR DESCRIPTION
**Pull request by Becojo**
Update the trivy version used in the trivy-image, trivy-sbom and trivy-sbom-image modules. 

I changed the setup step in trivy-image and trivy-sbom-image to extract the digests to environment variables. This should make patching the version and digests a bit easier.

### CODE REVIEW

File: scanners/boostsecurityio/trivy-sbom-image/module.yaml [hunk 1]
```diff
@@ -13,23 +13,28 @@ config:
 setup:
   - name: download trivy
     environment:
-      VERSION: 0.34.0
+      VERSION: 0.37.2
+      LINUX_X86_64_SHA: dd3f37d35160574f80092532e09938db73ffbb5649351ec3514def03811faffe
+      LINUX_ARM64_SHA: affd606ff83dab465b1eaeae0b3be41c91988b3e4b2833cdf62ca7f4678075f1
+      MACOS_ARM64_SHA: 097f25bf2e57561efb5f1c3625c5d3f934a265644d24690cd333bc543071b7be
+
```
Comment (JonathanSerafini) [inline scanners/boostsecurityio/trivy-sbom-image/module.yaml:20]: two extra newlines added here

File: scanners/boostsecurityio/trivy-image/module.yaml [hunk 2]
```diff
@@ -12,25 +12,28 @@ config:
 
 
 setup:
-  - name: download snyk-cli
+  - name: download trivy
     environment:
-      VERSION: 0.32.1
+      VERSION: 0.37.2
```
Comment (JonathanSerafini) [inline scanners/boostsecurityio/trivy-image/module.yaml:17]: 0.37.3 just came out an hour ago, if we're upgrading do we want to go straight to taht ?

File: scanners/boostsecurityio/trivy-image/module.yaml [hunk 3]
```diff
@@ -45,6 +48,7 @@ setup:
       rm trivy.tgz
       chmod +x trivy
 
+
```
Comment (JonathanSerafini) [inline scanners/boostsecurityio/trivy-image/module.yaml:51]: an extra newline is here as well ?

File: scanners/boostsecurityio/trivy-image/module.yaml (hunk 2 above)
Comment (Becojo) [reply scanners/boostsecurityio/trivy-image/module.yaml]: let's go with latest

**Review by JonathanSerafini — APPROVED** [review]

**Pull request merged**